### ISSUE TITLE
Fix false positives for all scope disables in `--report-invalid-scope-disables`.

### ISSUE THREAD

**Issue by ota-meshi** [stylelint/stylelint#4717]
> Clearly describe the bug

Using `--report-invalid-scope-disables` from the CLI incorrectly reports all scope disable comments.

> Which rule, if any, is the bug related to?

`--report-invalid-scope-disables` option.

> What code is needed to reproduce the bug?

I have created a minimal repository.
https://github.com/ota-meshi/stylelint-report-invalid-scope-disables-test

- index.css

```css
/* stylelint-disable block-no-empty */
a {}

/* stylelint-disable indentation */
  a {}
```

> What stylelint configuration is needed to reproduce the bug?

- .stylelintrc.json

```json
{
  "rules": {
    "block-no-empty": true,
    "indentation": 2
  }
}
```

> Which version of stylelint are you using?

`13.3.3`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

`stylelint --report-invalid-scope-disables index.css`

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

No. Standard css.

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors did you get)?

```shell
index.css
unused rule: block-no-empty, start line: 1
unused rule: indentation, start line: 4
```

Probably the `--report-invalid-scope-disables` feature doesn't refer to the rules defined in the config in the file.

The following repository is for reproducing bugs.
https://github.com/ota-meshi/stylelint-report-invalid-scope-disables-test
